### PR TITLE
Enable search-v2 operator to deploy postgress with SSL

### PR DESCRIPTION
**Pull request by dislbenn**
Related issue: https://github.com/stolostron/backlog/issues/21297
Signed-off-by: dislbenn <lavontae.bennett@gmail.com>

### CODE REVIEW

File: controllers/create_pgconfigmap.go
Comment (dislbenn) [inline controllers/create_pgconfigmap.go:26]: We still need to determine the certificate and key that we intend on using.

File: controllers/create_pgconfigmap.go
Comment (jlpadilla) [inline controllers/create_pgconfigmap.go:25]: I don't think this "secret" field is used for anything.

File: controllers/create_pgconfigmap.go
Comment (dislbenn) [reply controllers/create_pgconfigmap.go]: Removed `acm-proxyserver` label from the configmap.

File: controllers/create_pgservice.go
Comment (dislbenn) [reply controllers/create_pgservice.go]: Replace the value with the const variable.

**Comment by dislbenn** [comment]
@jlpadilla are there any more changes required for this PR? If not, it should be good to approve and merge.

**Comment by jlpadilla** [comment]
/approve
/hold
@xrajesh Do you want to do a quick review before we merge?

**Review by xrajesh — APPROVED** [review]

File: controllers/common.go
Comment (jlpadilla) [reply controllers/common.go:20]: @xrajesh `gosec` flags the variable name "secret" as a security violation.
@dislbenn Can you add this explanation in the code comment?

Comment (dislbenn) [reply controllers/common.go:20]: @jlpadilla @xrajesh correct. In sonar, this will mark the secret as a security violation, which will cause the sonar job to fail. This comment will just ignore that as a false positive for us.

Comment (dislbenn) [reply controllers/common.go]: Added the comment for this flag.

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Pull request merged**
